### PR TITLE
[ADVAPP-476] - Enhance presentation of Product Administration Section

### PR DESCRIPTION
**Pull request by ankit-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-476

### Technical Description

> Enhance presentation of Product Administration Section
_______________________________________________

### CODE REVIEW

**Review by dgoetzit — CHANGES_REQUESTED** [review]
@ankit-canyon This looks great - let's just make a small adjustment to the navigation sort convention to align with what is already in place.

**Comment by ankit-canyon** [comment]
> @ankit-canyon This looks great - let's just make a small adjustment to the navigation sort convention to align with what is already in place.

Hi @dgoetzit 

Double digit navigation sort no added. 

Thank you.

File: app/Filament/Clusters/ServiceManagementAdministration.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ class ServiceManagementAdministration extends Cluster
 
     protected static ?string $navigationGroup = 'Product Administration';
 
-    protected static ?int $navigationSort = 6;
+    protected static ?int $navigationSort = 60;
```
Comment (dgoetzit) [inline app/Filament/Clusters/ServiceManagementAdministration.php:47]: It looks like we have a duplicated sort of 60 between this and `app/Filament/Clusters/AssetManagement.php`. Can you adjust this, and bump up every item after this one?

**Review by dgoetzit — CHANGES_REQUESTED** [review]

File: app/Filament/Clusters/ServiceManagementAdministration.php (hunk 1 above)
Comment (ankit-canyon) [reply app/Filament/Clusters/ServiceManagementAdministration.php]: > It looks like we have a duplicated sort of 60 between this and `app/Filament/Clusters/AssetManagement.php`. Can you adjust this, and bump up every item after this one?

HI,

Issue is fixed.

Thank you.

**Review by dgoetzit — APPROVED** [review]

**Pull request merged**